ssr-express/index.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+/**
+  * The transform middleware replaces the `res.write` method so that, instead of sending
+  * the content to the network, it is accumulated in a buffer. `res.end` is also replaced
+  * so that, when it is invoked, the buffered response is transformed with AMP-SSR and sent
+  * to the network.
+  */
+
+const DEFAULT_AMP_PREFIX = '/amp';
+/**
+ * Creates a new amp-server-side-rendering middleware, using the specified
+ * ampSSR and options.
+ *
+ * There are 2 parameters available for options:
+ * - skipTransform: a `function(url: string): boolean` indicating if the transformation
+ *                  should be applied to the url.
+ * - ampUrl: a `function(url: string): string` that receives a url to be transformed and
+ *           returns the equivalent AMP url.
+ *
+ * @param {*} ampSSR the ampSSR instance to be used by this transformer.
+ * @param {*} options an optional object containing custom configureations for
```

Comment:
configureations?

Reply:
Can you provide a type instead of `*`?

Reply:
Can you explicitly comment the options? I did it like this for runtime-version: https://github.com/ampproject/amp-toolbox/blob/master/runtime-version/lib/RuntimeVersion.js#L38

Reply:
done.